examples/cra-kitchen-sink/.storybook/config.js

```diff
@@ -20,3 +20,5 @@ function loadStories() {
 }
 
 configure(loadStories, module);
+
+export { getStorybook };
```

Comment:
how does exporting `getStorybook` help exactly? i'm already familiar with the function.